### PR TITLE
Use travis stages

### PR DESCRIPTION
**Pull request by tiziano88**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -11,34 +11,22 @@ cache:
...
-env:
-  - FORMAT_CHECKS=true
-  - BUILD_SERVER=true
-  - BUILD_EXAMPLES=true
-  - UNIT_TEST=true
-  - EXAMPLES_TEST=true
-
-script:
-  - echo "build --show_progress_rate_limit=5.0" >> .bazelrc
-  - set -o errexit
```
Comment (anghelcovici) [inline .travis.yml]: Is this not needed anymore?

Comment (tiziano88) [reply .travis.yml]: I am not entirely sure, so I added it back for now. @daviddrysdale can confirm whether it is actually needed.

Comment (daviddrysdale) [reply .travis.yml]: Not sure &ndash; you could check by taking these 2 lines out and:
 - seeing if the Travis log gets truncated (due to excessive Bazel output)
 - put in a deliberate failure and check the Travis job fails.

Comment (tiziano88) [reply .travis.yml]: I will submit as it is, and then try and remove it in a future PR.

**Pull request merged**